test/data/openapi3/coverage.yaml

```diff
@@ -0,0 +1,52 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: OpenAPI3 Coverage Test
+  description: A Sample file for coverage test
+servers:
+- url: https://github.com/interagent/committee/
+paths:
+  /posts:
+    get:
+      description: get post
+      responses:
+        '200':
+          description: success
+          content:
+            application/json:
+              schema:
+                type: object
+        '404':
```

Comment:
OpenAPI 3 support `default.
When status code wasn't matched defined, use `default` definithion.
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#responses-object-example

So please implement and test it.

Reply:
done via https://github.com/interagent/committee/pull/297/commits/0712eedd3eb1e433dd2434b46912d6199cd7b1bb